wiki/BBCode/en.md

```diff
@@ -1,414 +1,294 @@
 
-- Dedicated button: ![Italic button](img/italic.png)
-- Notes:
-  - Use sparingly. Overuse will weaken the effectiveness of the emphasize.
-- Syntax:
-  ```
-  [i] ... [/i]
-  ```
+Toolbar button: ![Italic button](img/italic.png)
 
 ### Underline
 
-**Draws a horizontal line underneath** the word(s) or an entire paragraph.
+```
+[u]text[/u]
+```
+
+The `[u]` tag is used to emphasize text by drawing a horizontal line underneath said text (i.e., underlining). The horizontal line that is drawn will be affected by other tags like bolding and italicizing.
 
-- Dedicated button: N/A
-- Notes:
-  - Use sparingly. Overuse may create discomfort and make the post harder to read.
-- Syntax:
-  ```
-  [u] ... [/u]
-  ```
+### Strikethrough
 
-### Strike
+```
+[strike]text[/strike]
+```
 
-*Also known as **strikethrough**.*
+*Note: "strikethrough" may also be known as "strike".*
 
-**Draws a horizontal line through the middle** of the word(s) or an entire paragraph.
+The `[strike]` tag is used to indicate a removal of previously included text through the use of a horizontal line that "crosses-out" the text (i.e., a "strikethrough").
 
-- Dedicated button: ![Strike button](img/strike.png)
-- Notes:
-  - Use sparingly. Overuse may create discomfort and make the post harder to read.
-- Syntax:
-  ```
-  [strike] ... [/strike]
-  ```
+Toolbar button: ![Strike button](img/strike.png)
 
 ### Colour
 
-*BBCode calls this "Color".*
+```
+[color=HEXCODE]text[/color]
+```
 
-**Adds colour** to some text.
+*For a list of all colour names, see [X11 color names](https://en.wikipedia.org/wiki/X11_color_names#Color_name_chart "Wikipedia")*
 
-- Dedicated buttons: N/A
-- Notes:
-  - The colour box is located on the right side of the text box.
-  - Use this tag sparingly. Overuse of this will create discomfort for the reader's eyes. Misuse of this will make your text illegible.
-  - If you choose a colour, make sure it is is contrast to the background so that anyone can read it.
-  - The `HEXCODE` argument should not use quotation marks.
-- Syntax:
-  - Where `HEXCODE` is a colour in hexadecimal or colour name.
-    - For hexadecimal, it must start with `#`, followed by 6 hexadecimal (0 - 9, A - F) characters.
-    - For colour name, it must be a valid html colour.
-  ```
-  [color=HEXCODE] ... [/color]
-  ```
+The `[color]` tag is used to stylise text through various types of web safe colours. The tag uses the HEX code format to specify the colour, although it can also be specified through HTML colour names like "red" or "green." To specify, replace the `HEXCODE` argument with a colour's corresponding HEX code or HTML name.
 
-For a list of colour names, see [X11 color names](https://en.wikipedia.org/wiki/X11_color_names#Color_name_chart).
+The aforementioned argument does not take quotation marks (`"`), and does not have a default colour. If no HEX code or HTML name is specified, or if quotation marks are used, the tag will not be parsed as a BBCode tag.
```

Comment:
all of the others with arguments like this one mention the inclusion/exclusion of quotation marks, as well as the original article, so I thought it'd be better to mention it rather than not.